packages/dataviews/src/field-types/email.tsx

```diff
@@ -1,15 +1,15 @@
 /**
  * WordPress dependencies
  */
-import { isEmail } from '@wordpress/url';
```

Comment:
There are some differences between this utility and how the HTML standard expect inputs of type email to be validated ([see](https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address)). I've updated this to follow the standard, which is how the UI components at `@wordpress/components` work as well.

One notable difference is that the standard (and the UI components) consider `a@b` a valid email, while the `isEmail` utility doesn't. If we wanted to keep using `isEmail`, we'd need to look at how the UI components's behavior can be overridden.